### ISSUE TITLE
Kernel panic on version 1.5.4 was/is ok on version 1.4.12

### ISSUE THREAD

**Issue by Gost-Norw** [crystalidea/macs-fan-control#279]
![Macs fan controll ctrl panel older version 1 4 12 bug was on 1 5 4  2020-02-01 kl  22 11 14](https://user-images.githubusercontent.com/60555438/73599116-51c04500-4540-11ea-9b4d-a1a3dd65956e.png)

**Comment by Gost-Norw** [comment]
<!-- 

<details>
Macs Fan Control 1.4.12
MacBookPro10,1
CPU: Intel(R) Core(TM) i7-3740QM CPU @ 2.70GHz [100]
GPU: Intel HD Graphics 4000, NVIDIA GeForce GT 650M [active]
macOS 10.15.3

Fans:
Left side-min{2160}curr{3149}max{5940}mode{2,TC1C,59,78}
Right side-min{2000}curr{2243}max{5499}mode{2,TC2C,61,82}


TempSensors:
TB0T (Battery Max) - 34.3984, 34.3984
TB1T (Battery Sensor 1) - 34.3984, 34.3984
TB2T (Battery Sensor 2) - 31.6992, 31.6992
TC0E () - 69.1562, 69.1562
TC0F () - 71.1172, 71.1172
TC0P (CPU Proximity) - 50.875, 50.875
TC1C (CPU Core 1) - 67, 67
TC2C (CPU Core 2) - 63, 63
TC3C (CPU Core 3) - 61, 61
TC4C (CPU Core 4) - 61, 61
TCGC (GPU PECI) - 66, 66
TCSA () - 65, 65
TCTD () - 0.246094, 0.246094
TCXC (CPU PECI) - 66.2812, 66.2812
TG0D (GPU Diode) - 62.375, 62.375
TG0P (GPU Proximity) - 57, 57
TG1D () - 63, 63
TG1F () - 63.4414, 63.4414
TG1d () - 57, 57
TH0A () - 64, 64
TH0B () - 64, 64
TH0V () - 40.7891, 40.7891
TH0a () - 64, 64
TH0b () - 64, 64
TH0c () - 64, 64
TH0x () - 64, 64
TM0P (Memory Proximity) - 52.125, 52.125
TM0S (Memory Slot) - 51.832, 51.832
TMBS () - 51, 51
TP0P () - 52.75, 52.75
TPCD (Platform Controller Hub Die) - 59, 59
TS0D () - 129, 129
TS0P () - 129, 129
TS1D () - 129, 129
TS1P () - 129, 129
TW0P (Airport Card) - 44.5, 44.5
Ta0P () - 43.375, 43.375
TaSP () - 41.6406, 41.6406
Th1H (Heatsink Proximity) - 43.375, 43.375
Th2H () - 54, 54
Ts0P (Palm Rest) - 30.75, 30.75
Ts0S () - 40.9531, 40.9531
Ts1S () - 41.5469, 41.5469


Disk drives:
APPLE SSD SD512E - 64

_________________________________________________________________________________

NOT VERIFIED !
Updated the program Macs fan control 31 jan, The program has worked fine on multiple Mac Pro machines for more than 6 months.
My newest most powerful Mac got problems a short time after I installed the most recent version, my Mac got very slow and (with Kernal panic).
I am running Avira antivirus, Crome web-browser and Libre office.
I restarted my Mac and closed the programs mentioned, no change on Kernal panic, % CPU kernel "load" 800 to 1052.
I tried with multiple local accounts, restarted again. I waited a couple of hours and restarted again, the Mac seemed to be OK…
But Macs fan control was now not active, the program Icon was changed to a mostly white and gray Icon (circle with line going through, upper left to lower right) and the program size was now zero bytes in size.
When I looked at the right klick "Info" menu on the program Icon,the user Everyone had no access.
I uploaded the most recent Macs fan control.Zip version 1.5.4 updated 13. January from
URL: https://www.crystalidea.com/macs-fan-control/download to Virustotal but all of the scanner programs found nothing wrong.
I am going back to a version downloaded 1 September 2019 (Macs Fan Control 1.4.12.zip).
Seems OK now…

____________________________________________________________________________

https://www.crystalidea.com/macs-fan-control/faq

Why do I have high CPU usage by kernel_task when using Macs Fan Control?

Thermal issues are causing kernel task to preemptively use up CPU to prevent overheating. Try putting fan(s) back to auto and if it doesn't help, try to reset the SMC.

______________________________________________________________

I have not done the support tips...found them later...

</details>

**Comment by Gost-Norw** [comment]
NOT VERIFIED !
Updated the program Macs fan control 31 jan, The program has worked fine on multiple Mac Pro machines for more than 6 months.
My newest most powerful Mac got problems a short time after I installed the most recent version, my Mac got very slow and (with Kernal panic).
I am running Avira antivirus, Crome web-browser and Libre office.
I restarted my Mac and closed the programs mentioned, no change on Kernal panic, % CPU kernel "load" 800 to 1052.
I tried with multiple local accounts, restarted again. I waited a couple of hours and restarted again, the Mac seemed to be OK…
But Macs fan control was now not active, the program Icon was changed to a mostly white and gray Icon (circle with line going through, upper left to lower right) and the program size was now zero bytes in size.
When I looked at the right klick "Info" menu on the program Icon,the user Everyone had no access.
I uploaded the most recent Macs fan control.Zip version 1.5.4 updated 13. January from
URL: https://www.crystalidea.com/macs-fan-control/download to Virustotal but all of the scanner programs found nothing wrong.
I am going back to a version downloaded 1 September 2019 (Macs Fan Control 1.4.12.zip).
Seems OK now…

_______________________________________________________

Support on URL:https://www.crystalidea.com/macs-fan-control/faq

Why do I have high CPU usage by kernel_task when using Macs Fan Control?

Thermal issues are causing kernel task to preemptively use up CPU to prevent overheating. Try putting fan(s) back to auto and if it doesn't help, try to reset the SMC.

__________________________________________________________
I saw this later.....not tried this tip !
____________________________________________________________

**Comment by kleuter** [comment]
Please try.

**Comment by Gost-Norw** [comment]
I have solved the problem by decreasing the lower temperature for when the fans increase their speed from 60 to 45 degree C.
 Maybe my Mac HW has gotten slightly worse at dissipating heat.
If the problem stays away for some time I might increase it to 50 degrees C.
It seems that my Macbook Pro 15" Retina has been close to its thermal limit as evaluated by my Fan control prog "Macs Fan Control".
It`s a bit strange that "Macs Fan Control" has Kernel panic as the only "solution".

I have observed that my Mac, when it goes out of Hibernating/sleep mode may stay at a fan speed of approximately 2600RPM but go into "Kernel panic", this problem hopefully have disappeared with the new lower temperature limit.

The change from "Macs Fan Control" version 1.4.12  to 1.5.4 seems not to be related to my problem.

Thank you Kleuter for input :-) (we tried to solve the problem simultaneously) maybe Crystalidea will improve its fan control program to increase fanspeed when it activates "kernel panic" ...
